### PR TITLE
script result

### PR DESCRIPTION
**Pull request by JohnnyJumper**
This is the result of running the update script from [ #53 ].

### CODE REVIEW

File: src/supported-keys/supported-keys.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (JohnnyJumper) [inline src/supported-keys/supported-keys.json:2]: Is this needed to be updated? or this is just a mirror for the version from package.json?

File: src/supported-keys/supported-keys.json [hunk 2]
```diff
@@ -995,6 +995,1171 @@
       "deprecatedKeyName": "dns_ZONEMD_ttl",
       "validationRegex": null,
       "deprecated": false
+    },
+    "crypto.ETH.version.BSC.address": {
+      "deprecatedKeyName":  null,
```
Comment (JohnnyJumper) [inline src/supported-keys/supported-keys.json:1000]: As we are trying to move away from deprecatedKeyName I think it is ok to let it be null

File: src/supported-keys/supported-keys.json [hunk 3]
```diff
@@ -995,6 +995,1171 @@
...
+      "validationRegex": null,
+      "deprecated": false
+    },
+    "crypto.LINK.version.MATIC.address": {
+      "deprecatedKeyName":  null,
+      "validationRegex": null,
+      "deprecated": false
+    },
+    "crypto.LINK.version.HARMONYSHARD0.address": {
```
Comment (JohnnyJumper) [inline src/supported-keys/supported-keys.json:1034]: Some of the chains names ended up quite long. like this one HARMONYSHARD0. What can be a better name for it?

Comment (DeRain) [reply src/supported-keys/supported-keys.json:1034]: @JohnnyJumper if you not sure - remove this coins till the better times

File: src/supported-keys/supported-keys.json (hunk 2 above)
Comment (DeRain) [reply src/supported-keys/supported-keys.json:1000]: @JohnnyJumper It's not, website frontend still relies on that

File: src/supported-keys/supported-keys.json [hunk 4]
```diff
@@ -995,6 +995,1171 @@
       "deprecatedKeyName": "dns_ZONEMD_ttl",
       "validationRegex": null,
       "deprecated": false
+    },
+    "crypto.ETH.version.BSC.address": {
```
Comment (DeRain) [inline src/supported-keys/supported-keys.json:999]: BSC means Binance Smart Chain. I assume this is the BEP20 token, but you need to check

File: src/supported-keys/supported-keys.json [hunk 5]
```diff
@@ -995,6 +995,1171 @@
...
+      "deprecatedKeyName":  null,
+      "validationRegex": null,
+      "deprecated": false
+    },
+    "crypto.TUSD.version.HARMONYSHARD0.address": {
+      "deprecatedKeyName":  null,
+      "validationRegex": null,
+      "deprecated": false
+    },
+    "crypto.TUSD.version.BINANCECOIN.address": {
```
Comment (DeRain) [inline src/supported-keys/supported-keys.json:1449]: @JohnnyJumper Probably this is Binance Chain which means that the token might be BEP2 version. Please check.

**Review by DeRain — CHANGES_REQUESTED** [review]

File: src/supported-keys/supported-keys.json (hunk 1 above)
Comment (DeRain) [reply src/supported-keys/supported-keys.json:2]: Yes

**Review by DeRain — COMMENTED** [review]
@JohnnyJumper  Looks great! How much unique tickers was added (not considering versions)? If lesser than 200 it makes sense include more. The goal is to get 200 unique tickers.

**Pull request merged**